cypress/integration/1228-landing_page/anonymous_user_views_landing_page_for_registration/view_landing_page.js

```diff
@@ -1,15 +1,17 @@
-import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps';
-import { LANDING_PAGE_FIELDS } from '../../../support/data_testid_constants';
+import { LANDING_PAGE_FIELDS, LANDING_PAGE_SHARE_BUTTONS } from '../../../support/data_testid_constants';
 
 // @881
 // Scenario: Anonymous User views Landing Page for Registration
 When('an Anonymous user navigates to a Landing Page for a Resource', () => {
   cy.visit('/');
+  cy.setLocalStorage('beta', true); // TODO remove when sharing buttons are out of beta
   cy.get('[data-testid=search-button]').click();
   cy.get('[data-testid=search-field]').type('View Landing Page{enter}');
-  cy.get('[data-testid=result-list-item]').first().within((result) => {
-    cy.wrap(result).get('a').filter(':contains("View Landing Page")').click();
-  });
+  cy.get('[data-testid=result-list-item]')
+    .first()
+    .within((result) => {
+      cy.wrap(result).get('a').filter(':contains("View Landing Page")').click();
+    });

```

Comment:
Dette gjelder vel alle treff, så dette er kanskje godt nok?
```suggestion
  cy.get('[data-testid=result-list-item]').first().click();
```